### DISCUSSION TITLE
Sound whilst using Move Hub

### DISCUSSION THREAD

**Discussion by Monchichack** [pybricks/support#1892] [Q&A]
I've built Lego Frankie the cat which uses the Move hub and can't seem to work out how to add sound to the model using programming blocks. Any ideas?

**Comment by BertLindeman** [comment]
From the LEGO hubs I know, only the Spike Primehub and the Robot Inventorhub have a speaker.

The MoveHub does not support beep or speaker functions.

An advanced function might be to broadcast commands for the sound and observe that transmission on a PrimeHub and make the requested sound there.

Bert

**Comment by Monchichack** [comment]
Hi Bert,

In the Spike Prime system, a sound is recorded & saved on the pc, when played back it is played through the pc speakers.

On the boost app, the sound is recorded on the android device & when played back it is played through the android speakers.

This feature really brings a model like Frankie the cat alive, huge disappointment to find it missing after paying £50 for the Pybricks licence.

Chris

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by laurensvalk** [reply]
Hi Chris,

Thanks for the feedback. The [hub comparison table](https://pybricks.com/learn/getting-started/what-do-you-need/#choosing-a-lego-hub) says that the hub has no speaker, but it does indeed not say that you cannot play sounds in some other way.

So I think we should improve that page to make this clearer, and I'll be happy to refund your purchase.

Best,

Laurens

**Comment by Monchichack** [comment]
Hi Laurens,

Thanks for your support, I'd be grateful for a refund. Please let me know what information you require from me.

Kind Regards,
Chris

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by laurensvalk** [reply]
Thanks Chris, this has now been processed!

We do our best to create a fun additional play experience even as LEGO discontinues their various apps (at least one of the Boost apps is already gone), but Pybricks doesn't have all of the same features to be a complete replacement for all possible use cases. We might consider adding "remote sounds" at some point in a future version.
